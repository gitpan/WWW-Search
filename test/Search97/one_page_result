http://www.imf.org/search97cgi/s97_cgi.exe?QueryZip=burundi&ResultTemplate=imfrslp%2Ehts&QueryText=burundi&Collection=econiss&Collection=insext&Collection=external&SortOrder=Desc&ScoreThreshold=14&ResultStart=1&ResultCount=40&ServerKey=&AdminImagePath=&Theme=&Company=
