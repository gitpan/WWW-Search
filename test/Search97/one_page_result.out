http://www.imf.org/external/np/hipc/review/hipcres.pdf
http://www.imf.org/external/np/hipc/review/hipcref.pdf
http://www.imf.org/external/np/hipc/review/hipcrev.pdf
http://www.imf.org/external/pubs/ft/pam/pam52/pamphlet52.pdf
http://www.imf.org/external/pubs/ft/weo/weo0598/pdf/0598sta.pdf
http://www.imf.org/external/np/sec/decdo/wb.htm
http://www.imf.org/external/pubs/ft/ar/97/S97/pdf/file13.pdf
http://www.imf.org/external/pubs/ft/ar/97/F97/pdf/file13f.pdf
http://www.imf.org/external/pubs/ft/pam/pam51/pam51.pdf
http://www.imf.org/external/pubs/ft/ar/97/pdf/file13.pdf
http://www.imf.org/external/pubs/ft/ar/98/pdf/file08.pdf
http://www.imf.org/external/pubs/ft/ar/97/S97/pdf/file16.pdf
http://www.imf.org/external/pubs/ft/ar/97/F97/pdf/file16f.pdf
http://www.imf.org/external/pubs/ft/ar/97/pdf/file16.pdf
http://www.imf.org/external/pubs/ft/survey/pdf/072197.pdf
http://www.imf.org/external/pubs/ft/weo/weo1098/pdf/part6.pdf
http://www.imf.org/external/pubs/ft/ar/97/S97/pdf/file18.pdf
http://www.imf.org/external/pubs/ft/ar/97/F97/pdf/file18f.pdf
http://www.imf.org/external/pubs/ft/survey/sup0996/pdf/suppleme.pdf
http://www.imf.org/external/pubs/ft/survey/pdf/090096.pdf
http://www.imf.org/external/pubs/ft/ar/97/pdf/file18.pdf
http://www.imf.org/external/pubs/ft/pam/pam45/pamph452.pdf
http://www.imf.org/external/pubs/ft/survey/sup0997/pdf/suppleme.pdf
http://www.imf.org/external/pubs/ft/ar/98/pdf/file10.pdf
http://www.imf.org/external/pubs/ft/ar/98/pdf/file09.pdf
http://www.imf.org/external/pubs/ft/pam/pam52/3.htm
http://www.imf.org/external/np/sec/decdo/wfp.htm
http://www.imf.org/external/pubs/ft/Issues13/Issue13.pdf
http://www.imf.org/external/pubs/ft/Issues13/index.htm
http://www.imf.org/external/np/ins/english/training/hemail.htm
http://www.imf.org/external/pubs/ft/fandd/1996/03/pdf/alexande.pdf
http://www.imf.org/external/pubs/ft/fandd/1997/09/pdf/imf.pdf
http://www.imf.org/external/pubs/ft/ar/97/S97/pdf/file14.pdf
http://www.imf.org/external/pubs/ft/ar/97/F97/pdf/file14f.pdf
http://www.imf.org/external/pubs/ft/ar/97/pdf/file14.pdf
http://www.imf.org/external/pubs/ft/survey/pdf/080597.pdf
http://www.imf.org/external/np/hipc/pdf/ugancom.pdf
http://www.imf.org/external/pubs/ft/survey/pdf/011298.pdf
http://www.imf.org/external/pubs/ft/survey/pdf/020998.pdf
http://www.imf.org/external/pubs/pubs/cat82.pdf
